src/move/emojicoin_dot_fun/sources/emojicoin_dot_fun_rewards.move

```diff
@@ -0,0 +1,69 @@
+module emojicoin_dot_fun::emojicoin_dot_fun_rewards {
+    use std::signer;
+    use aptos_framework::account::{Self, SignerCapability};
+    use aptos_framework::aptos_account;
+    use aptos_framework::aptos_coin::AptosCoin;
+    use aptos_framework::coin;
+    friend emojicoin_dot_fun::emojicoin_dot_fun;
+
+    /// Resource account address seed for the vault.
+    const VAULT_NAME: vector<u8> = b"Vault";
+
+    struct RewardsVaultSignerCapability has key {
+        signer_capability: SignerCapability,
+    }
+
+    struct EmojicoinDotFunRewards has key {
+        apt_fees_paid: u64,
+        apt_rewards_claimed: u64
+    }
+
+    public entry fun claim(user: &signer)
+    acquires EmojicoinDotFunRewards, RewardsVaultSignerCapability {
```

Comment:
```suggestion
      acquires EmojicoinDotFunRewards, RewardsVaultSignerCapability {
```

nit, but I think this is the convention for multi-line breakage, right? Maybe it's different in Move..

Reply:
How about revisions per 2f17937?